packages/fxa-auth-server/lib/senders/emails/partials/paymentProvider/en.ftl

```diff
@@ -0,0 +1,8 @@
+payment-method = Payment Method:
```

Comment:
Ha, I was pondering this one too! I was stuck on a failing test until I noticed I had "View invoice:" instead of "View Invoice:", so I chose to remain consistent and capitalize.